src/components/Layout/Layout.tsx

```diff
@@ -148,7 +149,7 @@ export const Layout = forwardRef(function Layout(
                   <IconDoubleChevron aria-hidden="true" />
                   Menu
                 </Button>
-                [ Platform switcher goes here]
+                <PlatformNavigator currentPlatform={"Javascript"}/>
```

Comment:
Does using `currentPlatform` ([line 57ish](https://github.com/aws-amplify/docs/blob/5dd40aa515c702dfa92b343ddaad10af721bf959/src/components/Layout/Layout.tsx#L57)) for your currentPlatform prop work here?

Reply:
Yes this has been updated, sorry I should have left a comment here that the logic around the selected platform is still being worked on and will come as a follow up PR.  So for now this value is just a placeholder and will be updated once more once the platform logic is in place.